intel_extension_for_transformers/neural_chat/examples/deployment/textbot/backend/xeon/textbot.yaml

```diff
@@ -23,7 +23,7 @@
 host: 0.0.0.0
 port: 8000
 
-model_name_or_path: "./Llama-2-7b-chat-hf"
```

Comment:
should use neuralchatv3-1?

Reply:
This example uses the llm runtime int4 optimization. Will update it if the int4 issue is fixed.